blog/2025-07-30-dtg-july-update/index.md

```diff
@@ -0,0 +1,84 @@
+---
+slug: devtoolsguild-july-update
+title: Dev Tools Guild July update
+authors: [abcoathup]
+tags: [update]
+draft: true
+---
+
+# Dev Tools Guild July update
+
+_**TL;DR**: Ethereum and Solidity turn ten.  Ethereum Foundation, Optimism's Retro Funding and Octant support various member projects.  Gas limit to 45M_
+
+<!-- truncate -->
+
+## Dev Tools Guild members
+
+* [Argot Collective](https://argot.org/) (includes guild members Solidity & Sourcify) secured [three years of operational funding](https://argot.org/blog/ef-grant-announcement) from the Ethereum Foundation.
+* Optimism's [Retro Funding: Dev Tooling](https://atlas.optimism.io/missions/retro-funding-dev-tooling) (until end of 2025) is supporting member projects: [Solidity](https://atlas.optimism.io/project/0xcc8d03e014e121d10602eeff729b755d5dc6a317df0d6302c8a9d3b5424aaba8), [Vyper](https://atlas.optimism.io/project/0x9ca1f7b0e0d10d3bd2619e51a54f2e4175e029c87a2944cf1ebc89164ba77ea0), [alloy](https://atlas.optimism.io/project/0x56ce7cbc27852a8d8ef5869dc9033a215c8893f799468f61527dacb9f92be790), [ethers.js](https://atlas.optimism.io/project/0xa3d07f453f70d844196d89d79848aa2e70a0bd8b38bf0f493cba1547bb3bca5e), [Nethereum](https://atlas.optimism.io/project/0x4a5e771af86cf1938056b43cddbf0018dca1376d578f631f7449fe10ac4958ed), [viem](https://atlas.optimism.io/project/0x6bd057da522918a4675396313ae33a2f2788a1ceeb3bd7ae228015e3eb317a7d), [web3.py](https://atlas.optimism.io/project/0xebe03c3d6d33cad60124b9b05ef6e2ff056293a1de3c5fa51dfbb90c86c14bf7), [Ape](https://atlas.optimism.io/project/0xa0b16714baef75d97ec07fd48eaf42e79df92fe2a3c2d725d2388ede587ea54c), [Foundry](https://atlas.optimism.io/project/0x4562c0630907577f433cad78c7e2cc03349d918b6c14ef982f11a2678f5999ad), [Scaffold-ETH](https://atlas.optimism.io/project/0x154a42e5ca88d7c2732fda74d6eb611057fc88dbe6f0ff3aae7b89c2cd1666ab) and [Sourcify](https://atlas.optimism.io/project/0x51cda5996ef1a2ccd8fcf4ee5792337695599454c83eb1218c3ad4388dcb5bf5)
```

Comment:
We're in here too (Ape Framework)
https://atlas.optimism.io/project/0xa0b16714baef75d97ec07fd48eaf42e79df92fe2a3c2d725d2388ede587ea54c